Limits.md

```diff
@@ -0,0 +1,101 @@
+Exempting a specific user from all limits (e.g. power user)|User-specific limit with neither maximum concurrent CPU nor specific machine concurrency limits set
+Applying higher limits to a specific user (e.g. power user)|User-specific limit with concurrent CPU and/or specific machine concurrency limits set
+
+## Limits Lifecycle and State Machine
+
+Limits are applied at the control plane (upstream) cluster level, for any and all downstream clusters (including default), as a "meta-scheduling" construct.  This means that jobs with limits applied will be in "held" state until concurrency constraints for that user, tenant, or groups of tenants can be met.  Once limited jobs meet concurrency limit constraints, they are "released" to proceed via the downstream scheduling technology (e.g. Kubernetes or Slurm).  Once released, it's up to the downstream scheduler to place and execute the job.  If infrastructure is not available, jobs will still queue.
+
+![Job lifecycle state machine when limits applied](limits_state_machine.svg)
+
+Released (but still queued) jobs still count toward concurrency constraints - e.g. if a 16 CPU job is released but queued downstream, and the user or tenant has a 20 concurrent CPU limit applied, a newly submitted 16 core job will be held for limits immediately after submission.
+
+Note that other holding conditions may still apply after jobs are determined to meet limit constraints and released, such as license-based queuing.  This is evaluated **after** concurrency limits.
+
+## User Interface
+
+Jobs queued for limits will appear to end users with a substatus that indicates this on the *Dashboard->Current* view.
+
+System administrators can see this substatus in job details in the *Administration->Jobs* view.
+
+## Per-cluster Concurrent CPU Restriction: mL
+
+The **mL** setting can be applied per cluster in the *Administration->Clusters* view.  A value greater than 0 will be used in *MIN()* calculations for concurrent CPU limits, when applied.  For example, if a specific job is governed by a 16 concurrent CPU limit, but submitted on a downstream cluster with an *mL* value of 8, the limit will be adjusted to 8 concurrent CPUs.  This value can be used to govern small downstream clusters without using specific per-machine limits, especially in situations where users submit jobs to multiple concurrent clusters.  For example, a team administrator may want to limit a given user (or entire team) to 128 concurrent CPUs, regardless of which downstream clusters jobs are submitted to.  Users may be limited to 8 concurrent CPUs on a specific small downstream cluster, but able to consume a further 120 concurrent CPUs on larger clusters simultaneously.
+
+**IMPORTANT NOTE:** *mL* values apply **only** when concurrent CPU limits apply; otherwise ordinary infrastructure capacity is the initial arbitrer of resource availability.  *mL* values apply in both system administrator and self-service limits which define a concurrent CPU limit.
```

Comment:
```suggestion
**IMPORTANT NOTE:** *mL* values apply **only** when concurrent CPU limits apply; otherwise ordinary infrastructure capacity is the initial arbiter of resource availability.  *mL* values apply in both system administrator and self-service limits which define a concurrent CPU limit.
```